### PR TITLE
Implement RFC13, move element pointer to element

### PR DESCRIPTION
**Pull request by kylef**
RFC13 - https://github.com/refractproject/rfcs/blob/master/text/0013-element-pointer.md

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ All notable changes to the Refract project will be documented in this file.
 
 - Moved Compact Refract out of base specification
 - Removes the concept of namespaces and uses profiles instead
+- `Element Pointer` has become an element itself instead of an enum.
```
Comment (pksunkara) [inline CHANGELOG.md:10]: We already have 0.7.0 IIRC. That wasn't updated in changelog though.

File: refract-spec.md [hunk 2]
```diff
@@ -401,15 +401,16 @@ A pointer is an object for providing URLs to local elements and remote elements
...
 
-#### Members
+#### Properties
 
-- (string) - A URL to an ID of an element in the current document
-- (object)  - The URL and path of the referenced element
-    - `href` (string, required) - URL or ID of element
+- element: elementPointer (fixed)
+- attributes
     - `path` (enum) - Path of referenced element to transclude instead of element itself
```
Comment (pksunkara) [inline refract-spec.md]: Can you mark it as `string` here?

File: refract-spec.md [hunk 3]
```diff
@@ -401,15 +401,16 @@ A pointer is an object for providing URLs to local elements and remote elements
 1. When `path` is used, it references the given property of the referenced element
 1. When `path` is used in an element that includes the data of the pointer (such as with `ref`), the referenced path MAY need to be converted to a refract structure in order to be valid
 
-#### Members
+#### Properties
 
-- (string) - A URL to an ID of an element in the current document
-- (object)  - The URL and path of the referenced element
-    - `href` (string, required) - URL or ID of element
+- element: elementPointer (fixed)
```
Comment (pksunkara) [inline refract-spec.md]: Can you mark it as `string` here?

**Review by pksunkara — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (kylef) [reply CHANGELOG.md:10]: There is no tag for 0.7.0 so there is no point of reference for what is contained in 0.7.0.

Comment (pksunkara) [reply CHANGELOG.md:10]: The current master is 0.7.0

Comment (pksunkara) [reply CHANGELOG.md:10]: I will create the tag

Comment (pksunkara) [reply CHANGELOG.md:10]: The version number was in the README

Comment (kylef) [reply CHANGELOG.md:10]: > I will create the tag

The version in CHANGELOG should be updated too first.

**Review by pksunkara — APPROVED** [review]

**Pull request merged**